client/templates/index.html

```diff
@@ -1,5 +1,6 @@
 <head>
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <script src="//cdn.ckeditor.com/4.6.2/standard/ckeditor.js"></script>
```

Comment:
Are you sure this link is working?It should be https://cdn.ckeditor.com/4.6.2/standard/ckeditor.js